handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -256,6 +256,13 @@ public static boolean isAlpnSupported() {
     }
 
     /**
+     * Returns {@code true} if the used version of OpenSSL supports OCSP stapling.
+     */
+    public static boolean isOcspSupported() {
+      return version() >= 0x10002000L;
```

Comment:
Agreed I think we have the tools we need and everything is ok :) @rkapsi we just need to adjust the macros see https://github.com/netty/netty-tcnative/pull/215#discussion_r105228059 ... also also add more details here.